infra/core/managed-cluster/aks-cluster.bicep

```diff
@@ -54,6 +59,48 @@ module managedIdentity 'br/SharedDefraRegistry:managed-identity.user-assigned-id
   }
 }
 
+module managedIdentityAso 'br/SharedDefraRegistry:managed-identity.user-assigned-identity:0.4.3' = {
+  name: 'aso-managed-identity-${deploymentDate}'
+  params: {
+    name: asoPlatformManagedIdentity
+    tags: union(tags, tagsAsoMi)
+    location: location
+    lock: 'CanNotDelete'
+    federatedIdentityCredentials: [
+      {
+        name: '${asoPlatformManagedIdentity}-fic'
+        audiences: [
+          'api://AzureADTokenExchange'
+        ]
+        issuer: deployAKS.outputs.oidcIssuerUrl
+        subject: 'system:serviceaccount:azureserviceoperator-system:azureserviceoperator-default'
```

Comment:
NOTE: It is recommended that the default service account has no privileges. And each named service account used observes the least privilege model.

https://github.com/fluxcd/flux2-multi-tenancy#enforce-tenant-isolation

Should we be using a non default serviceAccount, or is it not relevant here?

Reply:
I'll take a look.  I'm not sure if it's possible to have a non-default azure service operator service account different to the 'azureserviceoperator-default' sa account.